dts/arm/nucleo_f091rc.dts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2017 Bobby Noelte
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/stm32f091.dtsi>
+
+/ {
+	model = "STMicroelectronics STM32F091RC-NUCLEO board";
+	compatible = "st,stm32f091rc-nucleo", "st,stm32f091";
+
+	chosen {
+		zephyr,console = &usart2;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+};
+
+&usart1 {
+	current-speed = <115200>;
+	pinctrl-0 = <&usart1_pins_a>;
+	pinctrl-names = "default";
+	status = "ok";
```

Comment:
USART1 is disabled. We should better remove 'status = "ok";'.

Reply:
Done